src/InstrumentationEngine/ProfilerManagerWrapper.h

```diff
@@ -0,0 +1,129 @@
+
+        // By default, logging messages are written to the debug output port. However,
+        // hosts can optionally signup to receive them through an instance of IProfilerManagerLoggingHost
+        STDMETHOD(SetLoggingHost)(_In_opt_ IProfilerManagerLoggingHost* pLoggingHost);
+
+        STDMETHOD(GetAppDomainCollection)(_Out_ IAppDomainCollection** ppAppDomainCollection);
+
+        STDMETHOD(CreateSignatureBuilder)(_Out_ ISignatureBuilder ** ppSignatureBuilder);
+
+        // Query the profiler manager for a pointer to another instrumentation method.
+        // Consumers can call QueryInterface to obtain an interface specific to that
+        // instrumentation method
+        STDMETHOD(GetInstrumentationMethod)(_In_ REFGUID cslid, _Out_ IUnknown** ppUnknown);
+
+        STDMETHOD(RemoveInstrumentationMethod(_In_ IInstrumentationMethod* pInstrumentationMethod));
+
+        // Registers a new instrumentation method in the profiler manager. Also calls its Initialize() method.
+        STDMETHOD(AddInstrumentationMethod)(_In_ BSTR bstrModulePath, _In_ BSTR bstrName, _In_ BSTR bstrDescription, _In_ BSTR bstrModule, _In_ BSTR bstrClassGuid, _In_ DWORD dwPriority, _Out_ IInstrumentationMethod** ppInstrumentationMethod);
+
+        // IProfilerManager2 Methods
+    public:
+        STDMETHOD(DisableProfiling)();
+
+        STDMETHOD(ApplyMetadata)(_In_ IModuleInfo* pMethodInfo);
+
+        // IProfilerManager3 Methods
+    public:
+        STDMETHOD(GetApiVersion)(_Out_ DWORD* pApiVer);
+
+        // IProfilerManager4 Methods
+    public:
+        STDMETHOD(GetGlobalLoggingInstance)(_Out_ IProfilerManagerLogging** ppLogging);
+
+        // IProfilerManager5 Methods
+    public:
+        STDMETHOD(IsInstrumentationMethodRegistered)(_In_ REFGUID clsid, _Out_ BOOL* pfRegistered);
+
+        // IProfilerManagerLogging Methods
+    public:
+        // If trace logging in enabled in the profiler manager, trace messages are sent to the
+        // profiler manager through this function.
+        STDMETHOD(LogMessage)(_In_ const WCHAR* wszMessage);
+
+        // Errors detected during profiling will be sent to the host through this method
+        STDMETHOD(LogError)(_In_ const WCHAR* wszError);
+
+        // If instrumentation result tracing is enabled, the detailed results of each instrumented
+        // method will be sent to the profiler manager host through this method.
+        STDMETHOD(LogDumpMessage)(_In_ const WCHAR* wszMessage);
+
+        // Called to cause logging to be written to the debug output port (via DebugOutputString) as well
+        // as to the host.
+        STDMETHOD(EnableDiagnosticLogToDebugPort)(_In_ BOOL enable);
+
+        // Allows instrumentation methods and hosts to ask for the current logging flags
+        STDMETHOD(GetLoggingFlags)(_Out_ LoggingFlags* pLoggingFlags);
+
+        // Allows instrumentation methods and hosts to modify the current logging flags
```

Comment:
These comments were copy-pasted from CProfilerManager, I can clean them up

Reply:
I imagine most of these are just declaring what's on the the IProfilerManager* interfaces. If so, you shouldn't need to have these comments here (they'll just get out of sync).